_posts/2015-11-16-grace-hopper-tips.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Grace Hopper Celebration Tips
+tags: ghc grace hopper women 
+username: tash1207 
+---
+
+Last month, I attended my first Grace Hopper Celebration. Going into it, I felt totally lost and overwhelmed and wished that I could have found a helpful blog post from a previous Hopper about what to expect. So I'm writing this in the hopes that someone attending GHC 2016 might find it useful.
```

Comment:
Can you link to the site for this event. Would help with getting more info on the subject material